NHLBI_BioData_Catalyst/python/7_Harmonization_with_PICSURE.ipynb

```diff
@@ -657,385 +743,365 @@
     "\n",
-    "final_df = pd.concat([clean for clean in [clean_fhs, clean_whi, clean_mesa]], ignore_index=True)\n",
-    "final_df = final_df.replace({\"Pneumonia\": mappings, 'Orthopnea':mappings})\n",
-    "final_df.head()"
+    "print(orthopnea_results.orthopnea.value_counts())\n",
+    "print('\\n')\n",
+    "print(orthopnea_results.pneumonia.value_counts())"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "ec29fe42",
+   "id": "760d2c67",
    "metadata": {},
    "source": [
     "Now that the variables have been encoded in the same way, we can use this dataframe for analysis and visualizations. "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "5e8eadef",
+   "metadata": {},
+   "source": [
+    "### Visualize"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "b13c630b",
+   "id": "65a0c031",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# library\n",
-    "import matplotlib.pyplot as plt\n",
-    "from matplotlib_venn import venn2"
+    "final_df = orthopnea_results\n",
+    "final_df['orthopnea'] = final_df['orthopnea']. fillna('Unknown')\n",
+    "final_df['pneumonia'] = final_df['pneumonia']. fillna('Unknown')"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "53673049",
+   "id": "190c5cff",
    "metadata": {},
    "outputs": [],
    "source": [
     "# Filters to apply to the final_df to create figures\n",
-    "pneu_yes = final_df['Pneumonia'] == 'Yes'\n",
-    "pneu_no = final_df['Pneumonia'] == 'No'\n",
-    "ortho_yes = final_df['Orthopnea'] == 'Yes'\n",
-    "ortho_no = final_df['Orthopnea'] == 'No'\n",
-    "fhs = final_df['Dataset'] == 'FHS'\n",
-    "mesa = final_df['Dataset'] == 'MESA'\n",
-    "whi = final_df['Dataset'] == 'WHI'"
+    "pneu_yes = final_df['pneumonia'] == 'Yes'\n",
+    "pneu_no = final_df['pneumonia'] == 'No'\n",
+    "ortho_yes = final_df['orthopnea'] == 'Yes'\n",
+    "ortho_no = final_df['orthopnea'] == 'No'\n",
+    "fhs = final_df['study'] == 'fhs'\n",
+    "mesa = final_df['study'] == 'mesa'\n",
+    "whi = final_df['study'] == 'whi'"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "5b5717d4",
+   "id": "8c717760",
    "metadata": {},
    "outputs": [],
    "source": [
     "# FHS figure\n",
-    "pneu_count =  final_df[pneu_yes & ortho_no & fhs].count()['Patient ID']\n",
-    "ortho_count = final_df[pneu_no & ortho_yes & fhs].count()['Patient ID']\n",
-    "both_count =  final_df[pneu_yes & ortho_yes & fhs].count()['Patient ID']\n",
+    "pneu_count =  final_df[pneu_yes & ortho_no & fhs].count()['patient_id']\n",
+    "ortho_count = final_df[pneu_no & ortho_yes & fhs].count()['patient_id']\n",
+    "both_count =  final_df[pneu_yes & ortho_yes & fhs].count()['patient_id']\n",
     "venn2(subsets = (pneu_count, ortho_count, both_count), set_labels = ('Pneumonia', 'Orthopnea'))\n",
     "plt.title(\"FHS Dataset\")\n",
     "plt.show()\n",
```

Comment:
cell worked and graph displayed correctly, but a warning was produced:

/opt/conda/lib/python3.10/site-packages/matplotlib_venn/_venn2.py:50: UserWarning: Both circles have zero area
  warnings.warn("Both circles have zero area")